vpc/proto/vpc.proto

```diff
@@ -126,6 +126,8 @@ message AssignIPRequestV3 {
     bool idempotent = 13;
     // Jumbo: I want "default routes" with an MTU of "Jumbo"
     bool jumbo = 14;
+    bool burst = 15;
+    uint64 bandwidth = 16;
```

Comment:
what do you think about encoding the units in the name?

Reply:
This is actually a good question on what the unit actually is...

Reply:
Big +1 on including the units in the name.